api/src/main/java/org/apache/iceberg/catalog/Namespace.java

```diff
@@ -47,6 +49,20 @@ private Namespace(String[] levels) {
     this.levels = levels;
   }
 
+  private  Map<String, String> parameters = new HashMap<>();
```

Comment:
`Namespace` is an identifier for a namespace. I don't think it should contain metadata.

Also, the Hadoop implementation of `createNamespace` ignores this metadata.

Reply:
This also makes no sense in some contexts, like `loadNamespaceMetadata`.

Reply:
ok delete the parameters from Namespace.java